app/src/main/java/com/abhijitvalluri/android/fitnotifications/services/NLService.java

```diff
@@ -443,16 +375,122 @@ private boolean appNotificationScheduleActive(String appPackageName) {
 
         int startTime = appSelection.getStartTime();
         int stopTime = appSelection.getStopTime();
-        Date currDate = new Date();
 
         // Get current time
         Calendar cal = Calendar.getInstance();
-        cal.setTime(currDate);
         int hour = cal.get(Calendar.HOUR_OF_DAY);
         int minute = cal.get(Calendar.MINUTE);
-        int currTime = hour*60 + minute;
+        int currTime = hour * 60 + minute;
+
+        return ((currTime >= startTime) && (currTime <= stopTime));
```

Comment:
No, I don't think every time the minute ends in 9 we should consider equality. If a user wants the schedule to be from 8:00AM to 5:59PM, then we shouldn't show notifications at 5:59:04PM. 

I think the best way to handle this is to (a) add a toggle that sets the schedule to "all day". In the database, this could be stored as 12:00AM to 11:59PM. And when this turns out to be case we can return true.

But let us handle this in the future and not as part of this review.